e2e/test/TwinFaultInjectionTests.cs

```diff
@@ -199,7 +199,10 @@ private async Task Twin_DeviceReportedPropertiesRecovery(Client.TransportType tr
 
                 await deviceClient.UpdateReportedPropertiesAsync(props).ConfigureAwait(false);
 
-                var deviceTwin = await deviceClient.GetTwinAsync().ConfigureAwait(false);
+                Twin deviceTwin = await deviceClient.GetTwinAsync().ConfigureAwait(false);
```

Comment:
>Twin [](start = 16, length = 4)

Just curious: The type needed for the Assert?

Reply:
Some testing found a null ref within the final assert line. This is to get more information of what exactly was null.